### PR TITLE
feat: radix theme

### PR DESCRIPTION
**Pull request by mortennordseth**
- Added @radix-ui/themes package to the project
- Styled the current pages to match the design sketches
- Note: Colors slightly differ from the design sketches, but this has been verified with the designer (colors in Figma do not exist in the Radix theme library).
- Future consideration: We might want to switch to Radix Primitives when we have progressed further with the design and would like to include our own styling, but we want to use the theme for now

Closes https://github.com/mrfylke/kontraktsportal-org/issues/36

<details>
<summary>Screenshots</summary>

image

image

image

image

</details>

### CODE REVIEW

**Comment by adriansberg** [comment]
Missing updated `package.json`?

File: src/components/header/header.module.css [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
   display: flex;
   justify-content: space-between;
-  padding: 1rem;
-}
-.header ul {
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
+  align-items: center;
+  gap: 0.5rem;
```
Comment (adriansberg) [inline src/components/header/header.module.css:5]: Does Radix have sizes that we'd want to use in the future?

**Comment by mortennordseth** [comment]
> Missing updated `package.json`?

Yep, forgot to push. Updated now

File: src/components/header/header.module.css (hunk 1 above)
Comment (mortennordseth) [reply src/components/header/header.module.css:5]: I don't think so. It looks like we are adding the AtB theme later

**Review by adriansberg — APPROVED** [review]

File: src/components/header/header.module.css (hunk 1 above)
Comment (mikaelbr) [reply src/components/header/header.module.css:5]: Where did you hear this? We talked about using some of the base styling, but I don't know if this means we should use the theming directly tbh.

**Pull request merged**